cmd/tools/gen-accessors.go

```diff
@@ -0,0 +1,300 @@
+// Copyright 2017 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
```

Comment:
We don't want `gen-accessors.go` in the same `github` directory because it is not in the `github` package and may cause confusion. It seems to be common practice to move command-line tools to a subdirectory called `tools` or `cmd` or `tools/cmd` or `cmd/tools`... I'm flexible with the directory name, but I think we want to keep it in a separate directory.